opintel/ckanext-mxtheme
# Modify text filter

### Changed Log: Changing text of filter of Datos en datos.gob.mx by resultados de búsqueda

*  ckanext/mxtheme/templates/snippets/search_result_text.html

### How to test: Follow the next steps:

1)  Go to your local environment to the url https://localhost/busca/dataset

* You have to see  resultado de búsqueda  instead of Datos en datos.gob.mx
